doc/guide/intro.md

```diff
@@ -1074,37 +1075,40 @@ An event is
 - the primitive events `never-evt` (bottom) and `always-evt` (top)
 - an event object, constructed with `make-event` or `wrap-evt`
 - an event-set object, constructed with `choice-evt`
-- an event-handler object, constructed with `handle-evt`; it is an event tied with a continuation function which is tail invoked with the value of the event. Multiple continuations can be chained with `handle-evt` each receiving the values of the previous, starting with the value of the vent.
+- an event-handler object, constructed with `handle-evt`; it is an event tied with a continuation function which is tail invoked with the value of the event. Multiple continuations can be chained with `handle-evt` each receiving the values of the previous, starting with the value of the event.
 
-`sync` accepts an arbitrary of events as arguments, and returns when exactly one of them is
+`sync` accepts an arbitrary number of events as arguments, and returns when exactly one of them is
 ready. The value of sync is the value of the event: by default, timeouts have a value of #f
 and other events have usually the synchronizer as value.
 
 
 For example:
 ```scheme
-(def my-thread (spawn (lambda () (thread-sleep! 10))))
-> (sync 1 my-thread)
+> (import :std/event)
+> (def (sleeping-thread t)
+    (spawn (lambda () (thread-sleep! t) 'done)))
+> (sync 1 (sleeping-thread 5)))
```

Comment:
This errors for me: `*** ERROR IN (console)@52.29 -- Datum or EOF expected`

Reply:
Should clarify: `(sync 1 (sleeping-thread 5)))` has an additional closing paren.

Reply:
Fixed, thanks.